Permissions.md

```diff
@@ -0,0 +1,127 @@
+LuckPerms has its own permissions for each command and various other features of the plugin. The permissions are listed under each command in the [Command Usage](Command-Usage) section, however, it may be useful to see the permissions together in a list of their own. You will find the list below.
```

Comment:
Shouldn't all those links be the full ones like it is everywhere else on the wiki?
Just for consistency.